tests/models/textnet/test_modeling_textnet.py

```diff
@@ -327,14 +327,18 @@ def test_inference_no_head(self):
         with torch.no_grad():
             output = model(**inputs)
 
-        # verify logits
-        self.assertEqual(output.logits.shape, torch.Size([1, 2]))
+        # verify output
+        self.assertEqual(output.last_hidden_state.shape, torch.Size([1, 512, 20, 27]))
         expected_slice_backbone = torch.tensor(
-            [0.9210, 0.6099, 0.0000, 0.0000, 0.0000, 0.0000, 3.2207, 2.6602, 1.8925, 0.0000],
+            [
+                [0.0000, 1.7415, 1.2660],
+                [0.0000, 1.0084, 1.9692],
+                [0.0000, 1.7464, 1.7892],
+            ],
             device=torch_device,
         )
         torch.testing.assert_close(
-            output.feature_maps[-1][0][10][12][:10], expected_slice_backbone, rtol=1e-3, atol=1e-3
+            output.last_hidden_state[0, 12, :3, :3], expected_slice_backbone, rtol=1e-2, atol=1e-2
         )
```

Comment:
So I guess `logits` and `feature_maps` doesn't make sense here, so we change to other things to check the values?

Reply:
There is no logit actually, it's either `last_hidden_state` for `TextModel` or `feature_maps` for `TextBackbone`